src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -4606,6 +4606,39 @@ BOOL MethodContext::repIsValidStringRef(CORINFO_MODULE_HANDLE module, unsigned m
     return value;
 }
 
+void MethodContext::recGetStringLength(CORINFO_MODULE_HANDLE module, unsigned metaTOK, int result)
+{
```

Comment:
I copy-pasted `IsValidStringRef` because it has the same signature (the only difference - `int` instead of `BOOL` for return type.